site/src/lib/api/blocks/slack.ts

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+
+import { mustGetEnvVar } from "../../../util/api";
+import { ExpandedBlockMetadata } from "../../blocks";
+import { isProduction } from "../../config";
+
+export const notifySlackAboutBlock = async (
+  block: ExpandedBlockMetadata,
+  changeType: "publish" | "update",
+) => {
+  if (!isProduction) return;
+
+  const webhookURL = mustGetEnvVar("SLACK_BLOCK_NOTIFICATION_WEBHOOK_URL");
```

Comment:
Hmm not sure why we need two checks here. We can configure `SLACK_BLOCK_NOTIFICATION_WEBHOOK_URL` in prod only, which will work as both. This will allow us to test the web hook locally if we want and also disable notifications in prod, if necessary.

Reply:
Hmm, so do you mean only enabling `production` for this specific env variable on Vercel?

Reply:
The app can only care about the presence of `process.env.SLACK_BLOCK_NOTIFICATION_WEBHOOK_URL`. If the variable is not set, notifications are silent. This gives us freedom to toggle the feature in prod only via this UI:
https://vercel.com/docs/concepts/projects/environment-variables#declare-an-environment-variable

Reply:
Updated in this commit, and also updated variable on Vercel https://github.com/blockprotocol/blockprotocol/pull/614/commits/68220b46969512447f72371c55d56a59e703e60c

Reply:
Cool! I guess we’ll also need to drop `mustGetEnvVar` as it will fail locally and in preview releases. You can use `envalid` instead with `default: ""` (you'll find an example in BP or HASH repo)